opus/application/apps/ui/test_ui.py

```diff
@@ -31,38 +32,60 @@ def tearDown(self):
 
 
             ###################################################
-            ######### api_last_blog_update UNIT TESTS #########
+            ######### api_notifications UNIT TESTS #########
             ###################################################
 
-    def test__api_last_blog_update_no_request(self):
-        "[test_ui.py] api_last_blog_update: no request"
+    def test__api_notifications_no_request(self):
+        "[test_ui.py] api_notifications: no request"
         with self.assertRaisesRegex(Http404,
-            r'Internal error \(No request was provided\) for /__lastblogupdate.json'):
-            api_last_blog_update(None)
+            r'Internal error \(No request was provided\) for /__notifications.json'):
+            api_notifications(None)
 
-    def test__api_last_blog_update_no_get(self):
-        "[test_ui.py] api_last_blog_update: no GET"
-        request = self.factory.get('__lastblogupdate.json')
+    def test__api_notifications_no_get(self):
+        "[test_ui.py] api_notifications: no GET"
+        request = self.factory.get('__notifications.json')
         request.GET = None
         with self.assertRaises(Http404):
-            api_last_blog_update(request)
+            api_notifications(request)
 
-    def test__api_last_blog_update_ok(self):
-        "[test_ui.py] api_last_blog_update: normal"
+    def test__api_notifications_default_ok(self):
+        "[test_ui.py] api_notifications: default - normal"
         settings.OPUS_LAST_BLOG_UPDATE_FILE = 'test_api/data/lastblogupdate.txt'
-        request = self.factory.get('__lastblogupdate.json')
-        ret = api_last_blog_update(request)
+        request = self.factory.get('__notifications.json')
+        ret = api_notifications(request)
         print(ret)
-        self.assertEqual(ret.content, b'{"lastupdate": "2019-JAN-01"}')
-
-    def test__api_last_blog_update_bad(self):
-        "[test_ui.py] api_last_blog_update: missing"
+        self.assertEqual(ret.content, b'{"lastupdate": "2019-JAN-01", "notification": null, "notification_mdate": null}')
+
+    def test__api_notifications_html_set(self):
+        "[test_ui.py] api_notifications: notification html set"
+        settings.OPUS_NOTIFICATION_FILE = "test_api/data/test_ui_notification.html"
+        request = self.factory.get('__notifications.json')
+        ret = api_notifications(request)
+        data = json.loads(ret.content)
+        print(data)
+        self.assertEqual(data['notification'], "<div><p>test</p></div>")
+        message = "Test value is none."
```

Comment:
There's no reason to include a message. The assert failing makes it obvious what the problem is.